include/libdnf/conf/option_string_list.hpp

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (C) 2018-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_CONF_OPTION_STRING_LIST_HPP
+#define LIBDNF_CONF_OPTION_STRING_LIST_HPP
+
+#include "option.hpp"
+
+#include <vector>
+
+namespace libdnf {
+
+/// Option that stores list of strings.
+/// It supports default value.
+/// It supports regular expression to check the values in the input list.
```

Comment:
It could be replaced by:
`/// Support default value, and check of an input value using the regular expression`